### PR TITLE
[github-actions-runner-operator] fix release namespace

### PR DESCRIPTION
**Pull request by dennybaa**

### CODE REVIEW

File: charts/github-actions-runner-operator/Chart.yaml [hunk 1]
```diff
@@ -30,11 +30,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.3.0
+version: 2.3.2
```
Comment (davidkarlsen) [inline charts/github-actions-runner-operator/Chart.yaml:33]:
```suggestion
version: 2.3.1
```

File: charts/github-actions-runner-operator/values.yaml [hunk 2]
```diff
@@ -24,6 +24,9 @@ serviceAccount:
 # limit to target namespace or cluster-wide
 clusterScoped: true
 
+## Set WATCH_NAMESPACE, when clusterScoped is false equls the release namespace
```
Comment (davidkarlsen) [inline charts/github-actions-runner-operator/values.yaml:27]:
```suggestion
## Set WATCH_NAMESPACE, when clusterScoped is false equals the release namespace
```

**Review by davidkarlsen — APPROVED** [review]

File: charts/github-actions-runner-operator/Chart.yaml (hunk 1 above)
Comment (dennybaa) [reply charts/github-actions-runner-operator/Chart.yaml]: hey, I'm well aware of this. of course it's just that I have actually made two releases and published in my repo)

**Comment by davidkarlsen** [comment]
thanks!

**Pull request merged**